CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @spocke @jhaines @metricjs @lnewson @ltrouton @tinydylan
```

Comment:
Did you want to include yourself in this list?

Reply:
not for the moment, no :)